tests/serializers/test_query_results.py

```diff
@@ -37,10 +37,14 @@ def test_doesnt_serialize_sensitive_keys_for_unauthenticated_users(self):
         self.assertSetEqual(set(["data", "retrieved_at"]), set(serialized.keys()))
 
 
-class CsvSerializationTest(BaseTestCase):
+class DsvSerializationTest(BaseTestCase):
     def get_csv_content(self):
```

Comment:
Could we avoid duplicating this function and just have something like `self.delimited_content("\t")`?

Reply:
@rauchy Thank you for review. I replaced two functions that `get_csv_content` and `get_tsv_content` with one function names `delimited_content` following your suggestion.

Actually I was worried if I would replace one when I added the test code.